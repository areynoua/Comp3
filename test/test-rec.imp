begin
  function @fib(i) do
    if i <= 1 then
      return i
    else
      j := @fib(i-1);
	  i := @fib(i-2);
	  j := i + j;
      return j
    endif
  end;
  read(a);
  (* b := @fib(a); *)
  print(b)
end
