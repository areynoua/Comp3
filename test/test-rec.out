Call to undefined function:
@fib is not a function name
